src/style.css

```diff
@@ -2031,7 +2033,11 @@ body.auto-camera #camera-recenter-tip {
 }
 
 #chatbox {
-  display: none;
+  /* hacky but we keep it displayed as a heightless block so that the transition works display: block; */
```

Comment:
Would like comments on this method, it may not be worth it to have chat transitions at all due to the hack required to get it to fade in and out through CSS

Reply:
I agree, I think it should just "pop in" (which is also what the other UI elements like the inventory do)